server/src/test/java/com/datastrato/gravitino/server/web/rest/TestMetalakeOperations.java

```diff
@@ -178,6 +183,19 @@ public void testCreateMetalake() {
     Assertions.assertEquals(ErrorConstants.ILLEGAL_ARGUMENTS_CODE, errorResponse.getCode());
     Assertions.assertEquals(
         IllegalArgumentException.class.getSimpleName(), errorResponse.getType());
+
+    String req2 = "{\"names\":\"metalake\",\"comment\":\"comment\",\"properties\":{}}";
```

Comment:
Could you also verify the other two exception mappers?

Reply:
Additionally, you can include comments on the illegal string to provide a brief explanation of why it is not allowed.

Reply:
> Could you also verify the other two exception mappers?

We can certainly verify `JsonParseExceptionMapper` with invalid Json request string with double `{{` such as `'{{"$names":"system","comment":"comment","properties":{}}'` 

However, `JsonProcessingExceptionMapper` is more tricky, in Jersey, only those `Exception`s do not belong to `JsonMappingException` or `JsonParseException` would trigger this mapper. For example `JsonGenerationException`, which is a subclass of `JsonProcessingException`, occurs when invalid Json writing, but not reading. Therefore it is hard to be generated by similar testing approach.

Reply:
Another example is `InputCoercionException`, which is support since jackson `2.10`, these kind of exception is generated when "failed attempts to convert specific Java value out of valid but incompatible input value", e.g. int too large. (Ref: https://fasterxml.github.io/jackson-core/javadoc/2.10/com/fasterxml/jackson/core/exc/InputCoercionException.html) This kind of exception seems also hard to create in metalake sinario.

Reply:
Thanks for your detailed feedback!

If so, it's recommended to register a new test path dedicated to testing the `JsonProcessingException`:
for example:
```
public class TestMetalakeOperations extends JerseyTest {
    // ....

    @Override
    protected Application configure() {
        // ......
        resourceConfig.register(MetalakeOperations.class);
        resourceConfig.register(TestException.class);
        // ......
    }
    
    @Path("/test")
    public static class TestException {
        @GET
        public Response getResponse() throws JsonProcessingException {
            throw new JsonProcessingException("Error processing JSON") {};
        }
    }
    
    // then test exception by /test

}
```

Reply:
I see, this approach looks great. 

Are we going to test other two mappers in original ways, or test them all in this test path. If all we want is to check whether the mapper is working properly, then the real invalid Json string was sent or not seems unimportant.

Reply:
suggest testing them all in this test path because we just want to know  whether mapper is working properly, instead of enumerating the incorrect JSON request

Reply:
@mchades The update has been push, please check the latest patch when you have time, thanks.